app/controllers/v1/books_controller.rb

```diff
@@ -1,37 +1,32 @@
+# frozen_string_literal: true
+
 module V1
   class BooksController < ApplicationController
     def reading_lists
-      books = current_user.books
+      reading_books = current_user.books
 
-      render json: renderer(books)
+      render_blueprint_api(
+        data: reading_books,
+        blueprint: Blueprinters::V1::BookBlueprint
+      )
```

Comment:
wouldn't assign this to a variable if we're going to use it once. Looks like we can just fit it in:
```rb
      render_blueprint_api(
        data: current_user.books,
        blueprint: Blueprinters::V1::BookBlueprint
      )
```

but then, it's a matter of preference